Tutorial compilation fails with error

When I follow the steps to compile the tutorial, I get the following error on the last make step:

```zsh
$  make tutorial
erl -config samples/nksip_tutorial/priv/app.config \
                -args_file samples/nksip_tutorial/priv/vm.args 
Erlang/OTP 17 [erts-6.4.1] [source] [64-bit] [smp:4:4] [async-threads:10] [kernel-poll:false]

Eshell V6.4.1  (abort with ^G)
(nksip_tutorial@127.0.0.1)1> 09:59:54.670 [error] CRASH REPORT Process <0.104.0> with 0 neighbours exited with reason: no match of right hand value {error,enoent} in nksip_config:make_cache/1 line 312 in application_master:init/4 line 133
{"Kernel pid terminated",application_controller,"{application_start_failure,nksip,{bad_return,{{nksip_app,start,[normal,[]]},{'EXIT',{{badmatch,{error,enoent}},[{nksip_config,make_cache,1,[{file,\"src/nksip_config.erl\"},{line,312}]},{nksip_config,make_cache,0,[{file,\"src/nksip_config.erl\"},{line,149}]},{nksip_app,start,2,[{file,\"src/nksip_app.erl\"},{line,60}]},{application_master,start_it_old,4,[{file,\"application_master.erl\"},{line,272}]}]}}}}}"}
Kernel pid terminated (application_controller) ({application_start_failure,nksip,{bad_return,{{nksip_app,start,[normal,[]]},{'EXIT',{{badmatch,{error,enoent}},[{nksip_config,make_cache,1,[{file,"src
make: *** [tutorial] Error 1

```

--- a/src/nksip_code_util.erl
+++ b/src/nksip_code_util.erl
@@ -172,6 +172,7 @@ compile(Mod, Tree) ->
 
 write(Mod, Tree) ->
     BasePath = nksip_config_cache:local_data_path(),
+    file:make_dir(BasePath),
     Path = filename:join(BasePath, atom_to_list(Mod)++".erl"),
     Content = list_to_binary(
         [io_lib:format("~s\n\n", [erl_prettypr:format(S)]) || S <-Tree]),

